client/lib/resource/src/create-resource/create-resource.js

```diff
@@ -10,13 +11,32 @@ class CreateResource extends BaseView {
         super({
             className: 'createResource',
             events: {
-                'submit': 'save'
+                'submit': 'save',
+                'keyup #resource_id': 'checkResourceIdAvailability'
             }
         });
         this.actions = Flux.getActions('resource');
     }
 
     /**
+     * Checks the resource store if a resource with this ID
```

Comment:
Ah, you're doing this here. Somehow thought it will be done on #66, but it's fine.